apps/nextjs-website/src/components/FramedVideo.tsx

```diff
@@ -0,0 +1,34 @@
+import { FramedVideo as FramedVideoRC } from '@react-components/components';
+import { FramedVideoProps } from '@react-components/types';
+import { FramedVideoSection } from '@/lib/fetch/types/PageSection';
+
+const makeFramedVideoProps = ({
+  videoURL,
+  video,
+  text,
+  sectionID,
+  theme,
+}: FramedVideoSection): FramedVideoProps => ({
+  sectionID,
+  theme,
+  videoURL: videoURL ?? '',
```

Comment:
```suggestion
  ...(videoURL && { videoURL }),
```
No fallback values, especially not values that turn an invalid value into a valid value. Null is NOT the same as ''.
If user did not input a value, we're not passing it through.